src/Tribe/Admin/Manager/Service_Provider.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Tribe\Tickets\Admin\Manager;
+
+use tad_DI52_ServiceProvider;
+
+/**
+ * Class Manager
+ *
+ * @package Tribe\Tickets\Admin\Manager
+ *
+ * @since   TBD
+ */
+class Service_Provider extends tad_DI52_ServiceProvider {
+	/**
+	 * Register the provider singletons.
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		$this->container->singleton( 'tickets.admin.manager', self::class );
+
+		$this->hooks();
+	}
+
+	/**
+	 * Add actions and filters.
+	 *
+	 * @since TBD
+	 */
+	protected function hooks() {
+		// Handle AJAX.
+		add_action( 'wp_ajax_nopriv_tribe_tickets_admin_manager', [ $this, 'ajax_handle_admin_manager' ] );
+		add_action( 'wp_ajax_tribe_tickets_admin_manager', [ $this, 'ajax_handle_admin_manager' ] );
+	}
+
+	/**
+	 * Handle response
+	 *
+	 * @since TBD
+	 */
+	public function ajax_handle_admin_manager() {
```

Comment:
At some point, we should have capabilities checks here. Add a Todo I think since it might be too early for that.

Reply:
```suggestion
	public function ajax_handle_admin_manager() {
		// @todo Look at adding capability checks of some sort based on a filter that provides capability context for the specific request.
```